dymos/utils/testing_utils.py

```diff
@@ -0,0 +1,45 @@
+import os
+import shutil
+import tempfile
+
+
+def _new_setup(self):
+    self.startdir = os.getcwd()
+    self.tempdir = tempfile.mkdtemp(prefix='testdir-')
+    os.chdir(self.tempdir)
+    if hasattr(self, 'original_setUp'):
+        self.original_setUp()
+
+
+def _new_teardown(self):
+    if hasattr(self, 'original_tearDown'):
+        self.original_setUp()
```

Comment:
this should call `self.original_tearDown`?

Reply:
Good catch